linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/bash/shared.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+SECURE_MAX_PASS_AGE=60
+
+usrs_max_pass_age=( $(awk -F: '$5 > SECURE_MAX_PASS_AGE || $5 == "" {print $1}' /etc/shadow) )
+for i in ${usrs_max_pass_age[@]};
+do
+  passwd -x $SECURE_MAX_PASS_AGE $i
```

Comment:
I see that the corresponding rule.yml says that you should run `chage` but in the remediation you run `passwd -x`. I think that the remediation and the rule.yml should be consistent, so I suggest changing either the remediation or the description in the rule.yml.